xdocs/usermanual/component_reference.xml

```diff
@@ -6491,11 +6491,15 @@ Selecting Functional Testing instructs JMeter to save the  additional sample inf
 This increases the resources needed to run a test, and may adversely impact JMeter performance.
 If more data is required for a particular sampler only, then add a Listener to it, and configure the fields as required.
 <note>The option does not affect CSV result files, which cannot currently store such information.</note>
+<note>For non gui test, you can set the Functional Testing with the jmeter property : jmeter.test_plan.functional_mode (true or false, false default value). This property is not by default in the jmeter.properties</note>
```

Comment:
Unlike in French typography, no space is put before the colon in English typography.
```suggestion
<note>For non-GUI test, you can set the Functional Testing mode with the JMeter property: <code>jmeter.test_plan.functional_mode</code> (<code>true</code> or <code>false</code>, default value is <code>false</code>). This property is not by default in the <code>jmeter.properties</code> file.</note>
```